tests/StlTests.cpp

```diff
@@ -151,6 +157,27 @@ TEST_CASE("StlTests::TestZStringView literal", "[stl][zstring_view]")
     }
 }
 
+#if defined(__cpp_lib_format) && (__cpp_lib_format > 201907L)
+
+TEST_CASE("StlTests::TestZStringView formatting", "[stl][zstring_view]")
+{
+    SECTION("zstring_view can be used with std::format(wchar_t const*)")
+    {
+        auto str = L"kittens"_zv;
+        auto f = std::format(L"Hello {}", str);
+        REQUIRE(f == L"Hello kittens");
+    }
+
+    SECTION("zstring_view can be used with std::format(char const*)")
+    {
+        auto str = "kittens"_zv;
+        auto f = std::format("Hello {}", str);
+        REQUIRE(f == "Hello kittens");
+    }
+}
+
+#endif // __WI_LIBCPP_STD_VER >= 20

```

Comment:
```suggestion
#endif
```
A good real-time example of why I dislike these types of comments